cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -537,30 +537,35 @@ type Readiness struct {
 	Unregistered int
 	// Time when the readiness was measured.
 	Time time.Time
+	// Number of nodes that are either Unready or NotStarted due to missing resources.
```

Comment:
Looking at implementation it seems to only count unready nodes and not notStarted? Also maybe it's worth mentioning that this is only used for reporting purposes and not in actual calculations?

Reply:
Good catch, thanks. That was a leftover from the initial implementation, which used to count notStarted too. I ultimately decided it is simpler to make it a subset of a single existing category. All notStarted nodes will change their state within 15m anyway.